### DISCUSSION TITLE
PP-ChatOCR 之前的开发者模式的源码

### DISCUSSION THREAD

**Discussion by sugarandgugu** [PaddlePaddle/PaddleOCR#13205] [Q&A] [answered]
之前使用PP-ChatOCR的demo，是不提供写入prompt的，需要进入到开发者模式进行修改。现在的页面没有开发者模式了，但是文档还没有更新。想请问一下还有没有别的方式拿到源码(SDK的封装方式过于复杂了)？
页面: https://aistudio.baidu.com/modelsdetail/332
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/95059219/99245305-75db-4ca3-ad51-6e9973990ac7)

**Comment by sugarandgugu** [comment]
有人吗

**Reply by GreatV** [reply, marked as answer]
这个不属于paddleocr项目的内容，你可能需要在ai studio上咨询。
